src/caselawclient/models/neutral_citation_mixin.py

```diff
@@ -40,14 +40,8 @@ def neutral_citation(self) -> str:
 
     @cached_property
     def has_ncn(self) -> bool:
-        if not self.neutral_citation:
-            return False
-
-        return True
+        return bool(self.neutral_citation)
 
     @cached_property
     def has_valid_ncn(self) -> bool:
-        if not self.has_ncn or not neutral_url(self.neutral_citation):
-            return False
-
-        return True
+        return self.has_ncn and neutral_url(self.neutral_citation) is not None
```

Comment:
I'm not convinced this is correct. 
Neutral citation is a string, so this should probably be `!= ""`.

This brings into doubt a pile of other simplifications...